### PR TITLE
mark server telemetyr channel with internal SDK operation

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Step toward https://github.com/Microsoft/ApplicationInsights-dotnet/issues/440

### CODE REVIEW

File: src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetrySerializer.cs [hunk 1]
```diff
@@ -8,8 +8,9 @@
 
     internal class TelemetrySerializer
     {
+        internal readonly Transmitter Transmitter;
```
Comment (dnduffy) [inline src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetrySerializer.cs]: nitpick: Why the capitalization change?  This isn't a property or a constant, does StyleCop want readonly variables to be capitalized also?

Comment (SergeyKanzhelev) [inline src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetrySerializer.cs]: Internal should be capitalized according to StyleCop

Comment (dnduffy) [inline src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetrySerializer.cs]: I think StyleCop is confused in this case. :(

Comment (SergeyKanzhelev) [inline src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetrySerializer.cs]: I'll be happy to follow any other ruleset. Keeping it compliant to the current ruleset for now.

File: src/Core/Managed/Shared/OperationTelemetryExtensions.cs [hunk 2]
```diff
@@ -40,7 +40,9 @@ public static void Start(this OperationTelemetry telemetry, long timestamp)
             // Stopwatch.GetTimestamp() is used (instead of ElapsedTicks) as it is thread-safe.
             telemetry.BeginTimeInTicks = timestamp;
 
+#if !CORE_PCL
```
Comment (pharring) [inline src/Core/Managed/Shared/OperationTelemetryExtensions.cs]: I just saw your IM. Thanks for fixing this. I didn't realize it had been broken for a while now. Yes, eventually we want RichPayloadEventSource in Core.PCL.

**Pull request merged**